virtual_workstation_setup.sh

```diff
@@ -155,10 +155,13 @@ setup_master() {
 
             python -m pip install "pip==21.0.1"
 
-            python -m pip install -r etc/pip/dev-requirements.txt
-            python -m pip install keyring
+            python -m pip install 'poetry==1.5.1'
+            # PYTHON_KEYRING_BACKEND is needed to make poetry install work
+            # See guide https://wiki.corp.mongodb.com/display/KERNEL/Virtual+Workstation
+            export PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring
+            python -m poetry install --no-root --sync
```

Comment:
Why not only expose it to the single command (both here and on the wiki)? That both makes it clear which command it is supposed to be modifying and avoids permanent environment pollution. A side benefit is that it makes it easer to re-run the command since it is a single entry in history that you can easily find with <kbd>CTRL</kbd>+<kbd>R</kbd>.

```bash
PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring python -m poetry install --no-root --sync
```

Reply:
Yea that is a good suggestion. Next time I touch this script I'll do your suggestion